src/main/java/org/dcsa/ovs/notifications/controller/NotificationEndpointController.java

```diff
@@ -0,0 +1,38 @@
+package org.dcsa.ovs.notifications.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.core.controller.ExtendedBaseController;
+import org.dcsa.ovs.notifications.model.NotificationEndpoint;
+import org.dcsa.ovs.notifications.service.NotificationEndpointService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.server.reactive.ServerHttpRequest;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "notification-endpoints", produces = {MediaType.APPLICATION_JSON_VALUE})
+public class NotificationEndpointController extends ExtendedBaseController<NotificationEndpointService, NotificationEndpoint, UUID> {

```

Comment:
I think we need to overwrite all CRUD operations - and return IllegalAccess (or whatever we do other places). It should not be possible to call GET /notification-endpoints, or DELETE or PUT

Reply:
Done